cmd/amp/account.go

```diff
@@ -186,6 +196,29 @@ func getCode() (code string) {
 	return
 }
 
+// pwdChange validates the input command line arguments and changes the current password
+// by invoking the corresponding rpc/storage method
+func pwdChange(amp *client.AMP, cmd *cobra.Command, args []string) error {
+	username := getUserName()
+	existingPwd := getExistingPwd()
+	newPwd := getPwd()
+	getConfirmNewPwd(newPwd)
+	request := &account.PasswordChangeRequest{
+		Username:         username,
+		ExistingPassword: existingPwd,
+		NewPassword:      newPwd,
+	}
+	client := account.NewAccountServiceClient(amp.Conn)
+	_, err := client.PasswordChange(context.Background(), request)
+	if err != nil {
+		return err
```

Comment:
`return fmt.Errorf("server error: %v", err)`

This should be consistent with the other commands.